pkg/simple/client/devops/jenkins/pure_request.go

```diff
@@ -24,10 +30,41 @@ func (j *Jenkins) SendPureRequestWithHeaderResp(path string, httpParameters *dev
 
 	client := &http.Client{Timeout: 30 * time.Second}
 
+	header := httpParameters.Header
+	bearTokenArray := strings.Split(header.Get("Authorization"), " ")
+	bearFlag := bearTokenArray[0]
```

Comment:
What if there is no `Authorization` header in the request? Better to check bearTokenArray first.

Reply:
Even if there is no Authorization header，bearTokenArray is assigned as [''] .